packages/design-system/src/Button/Button.stories.tsx

```diff
@@ -0,0 +1,163 @@
+      table: {
+        type: { summary: "boolean" },
+        defaultValue: { summary: "false" },
+      },
+    },
+    kind: {
+      control: {
+        type: "radio",
+        options: ["primary", "secondary", "tertiary", "error"],
+      },
+      description: "The visual style to apply to the button.",
+      table: {
+        type: { summary: "primary | secondary | tertiary | error" },
+        defaultValue: { summary: "primary" },
+      },
+    },
+    size: {
+      control: {
+        type: "radio",
+        options: ["sm", "md", "lg"],
+      },
+      description: "The size of the button.",
+      table: {
+        type: { summary: "sm | md | lg" },
+        defaultValue: { summary: "sm" },
+      },
+    },
+    startIcon: {
+      control: {
+        type: "text",
+      },
+      description:
+        "The icon to display before the button text. Pass name of the icon from remix-icon library(eg: home-2-line) or an svg icon.",
+      table: {
+        type: { summary: "React.ReactNode | string" },
+      },
+    },
+    endIcon: {
+      control: {
+        type: "text",
+      },
+      description:
+        "The icon to display after the button text. Pass name of the icon from remix-icon library(eg: home-2-line) or an svg icon.",
+      table: {
+        type: { summary: "React.ReactNode | string" },
+      },
+    },
+    UNSAFE_height: {
+      control: {
+        type: "text",
+      },
+      description: "The height of the button. Accepts all css units.",
+      table: {
+        type: { summary: "string" },
+      },
+    },
+    UNSAFE_width: {
+      control: {
+        type: "text",
+      },
+      description: "The width of the button. Accepts all css units.",
+      table: {
+        type: { summary: "string" },
+      },
+    },
+    isDisabled: {
+      control: {
+        type: "boolean",
+      },
+      description: "Whether the button is disabled.",
+      table: {
+        type: { summary: "boolean" },
+        defaultValue: { summary: "false" },
+      },
+    },
+    href: {
+      control: {
+        type: "text",
+      },
+      description:
+        "The href attribute to apply to the button if it renders as an anchor.",
+      table: {
+        type: { summary: "string" },
+      },
+    },
+    onPress: {
+      control: {
+        type: "function",
+      },
+      description: "The callback to invoke when the button is pressed.",
+      table: {
+        type: { summary: "function" },
+      },
+    },
+  },
+  parameters: {
+    docs: {
+      description: {
+        component:
+          "A button component with different variants and sizes. It can be used as a button or an anchor. Other than the defined props button component accepts all the props of the button and a elements. ",
+      },
+    },
+    controls: { sort: "requiredFirst" },
+  },
+} as ComponentMeta<typeof Button>;
+
+// eslint-disable-next-line react/function-component-definition
```

Comment:
We should try using CSF - 3.0 - https://storybook.js.org/blog/component-story-format-3-0, in the next storybook version the above format will be deprecated.

Reply:
Thank you @yaldram  for pointing this out.
cc: @tanvibhakta